WIP: Student-facing Teams Doc

catong opened the pull request:
Student-facing doc for teams.
@explorerleslie @andy-armstrong FYI -- still in progress as we get a complete sandbox. 
Draft HTML project for easier reading:
http://draft-student-teams.readthedocs.org/en/latest/SFD_teams.html

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+About Teams and Topics
+************************
+
+If teams are used in your course, there is a **Teams** page in the LMS where
```
Do we call them pages or tabs? Silly me, I've always called them tabs.

catong replied:
Me too, but we (the doc team) are trying to become more consistent ever since making a decision for our style guide that they should be called "pages" :-)

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+
+************************
+About Teams and Topics
```
I think people get a little tripped up about teams and topics -- they don't understand the relationship between the two, that instructors create topics for teams to form project or activity groups around, and students pick a topic they are interested in and then can join or create a team within a topic. Can we be really explicit about that relationship here?

explorerleslie replied:
It's almost like, we need to explain to students why their instructors created topics.

catong replied:
Yep, will expand. Thanks for the tip.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+If it up to you to browse the topics and find one that you are interested in
+working on. Then, you join a team that is working on that topic. For more
+information, see :ref:`Finding a Team to Join`.
```
In the HTML draft, this isn't showing up as a link for some reason.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+#. To get a better sense of a team's members, discussion, and communications,
+   select **View** for a team.
+
+   On a team's detail page, you can browse the team's discussion topics, but
```
Nit: it's discussion posts, not topics. Topics contain posts.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+Join a Team
+=======================
+
+When you have found a team you want to join, select **Join Team**.
```
do we want to mention that the team must have available slots, or the Join Team button won't appear?

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+.. image to come
+
+#. Forward the link to friends who you want to invite. They can paste the URL
```
Forward how? Email, social media.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+.. image to come
+
+.. Is there a confirmation message? Can't do on sandbox yet.
```
There is a confirmation message but it won't be implemented for next week.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+
+=======================
+Leaving a Team
```
We may want to say here something about that edX recommends that you don't leave your team high and dry in the middle of the course by leaving. If you are mid-work, maybe let your team know that you're leaving and why? something like that.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+=======================
+
+Only if you cannot find an appropriate team to join within your topic, you can
+create a new team. 
```
Not technically true...they CAN create a new team whenever they want. I would soften this message like you did either earlier or in the course author docs. edX recommends that you join an existing team wherever possible, and only create a new team if no teams exist, if there are no teams with open slots, or if you don't find one that interests you.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+Only if you cannot find an appropriate team to join within your topic, you can
+create a new team. 
+
+When you create a new team, you are automatically made a member of it.
```
nit: "made a member" is a weird phrase to me. Added as a member?

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+.. image to come  
+
+#. On the **Create New Team** page, add a name and description for the team.
```
Add something like, the description of your team should help other learners decide if they want to work on the project or activity you are proposing.

explorerleslie commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,174 @@
+
+#. When you have finished adding your team details, click **Create**.
+
+   Your new team is added to the list of team under your selected topic. You are automatically made a member of the team you created.
```
same thing about made a member

catong commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,213 @@
+their descriptions.
+
+It is up to you to browse the topics and find one that you are interested in
+working on. Then, you join a team that is working on that topic, either by
```
"join a team...either by joining a team" is a bit weird. I need to tweak this.

andy-armstrong commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,238 @@
+  :alt: The Leave Team link on the Team Details page.
+
+You are no longer visible in the team membership profiles list, or in the
+membership count. You can no longer add posts in the team's discussion.
```
"add posts or replies..."?

andy-armstrong commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,238 @@
+Creating a Team
+=======================
+
+If no teams exist in the topic, if existing teams have reached the maximum
```
I know that we are trying to discourage the creation of new teams but I found this a very confusing sentence to parse. You have to make it all the way through to the end to work out what the 'then' part is for the three ifs.

andy-armstrong commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,238 @@
+
+   Your new team is added to the list of team under your selected topic. You are automatically added as a team member.
+
+   For information about inviting friends to join a team, see :ref:`Inviting
```
Remove all the documentation around invitations.

andy-armstrong commented:
@catong There's something odd about the left-hand links in the Draft HTML documentation. When I click on them, the browser scrolls me down beyond the bottom of the list. I don't know if this is a known issue, or if it indicates an issue in the markup.

andy-armstrong commented on en_us/shared/students/SFD_teams.rst:
```diff
@@ -0,0 +1,238 @@
+   before you can join a new one.
+
+
+.. _Inviting Friends to Join Your Team:
```
Remove the invitation functionality.

andy-armstrong commented:
:+1: once the invitation material has been removed.

catong commented:
@andy-armstrong I've addressed your comments. 
FYI I looked into the TOC scrolling issue and it's unfortunately a "feature" of RTD ;-)  When you click on a heading in the left side TOC, the reading pane on the right scrolls to that topic, but the TOC scrolls together with the right pane as one. So if the heading you clicked on is far down on the current HTML page, you might be far down enough to have the TOC scroll up out of sight.

catong commented:
@symbolist Not sure who on your team should take a look?

symbolist commented:
@muzaffaryousaf Can you review this?

muzaffaryousaf commented:
@catong FYI, we are not supporting Invite Link functionality for now and it has been removed from UI.

catong commented:
@muzaffaryousaf Thank you for the reminder about the invitation feature. I had meant to remove that section.

muzaffaryousaf commented on en_us/shared/building_and_running_chapters/teams/teams_discussions.rst:
```diff
@@ -16,9 +16,12 @@ For information about discussion administration roles, see
 :ref:`Assigning_discussion_roles`. For information about managing discussions,
 see :ref:`Running_discussions`.
 
+<<<<<<< HEAD
```
@catong can you address this  bad rebase.

muzaffaryousaf commented on en_us/shared/building_and_running_chapters/teams/teams_setup.rst:
```diff
@@ -10,7 +10,11 @@ in your courses.
 
 .. contents::
   :local:
+<<<<<<< HEAD
```
This is in multiple files.

catong commented:
Closing to create new PR against master rather than against carol/doc/teams_CA branch.
